bears/python/DodgyBear.py

```diff
@@ -0,0 +1,37 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='dodgy',
+        use_stdout=True,
+        use_stderr=False,
+        global_bear=True)
+class DodgyBear:
+    """
+    Checks Python files for "dodgy" looking values such
+    as AWS secret keys, passwords, SCM diff check-ins,
+    SSH keys and any other type of hardcoded secrets.
+    """
+
+    LANGUAGES = {'Python'}
+    REQUIREMENTS = {PipRequirement('dodgy', '0.1.9')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Security', 'Hardcoded Secret',
+                  'SCM Diff Check-in', 'SSH Keys'}
+    SEE_MORE = 'https://github.com/landscapeio/dodgy'
+
+    @staticmethod
+    def create_arguments(config_file):
+        return []
+
+    def process_output(self, output, filename, file):
+        for issue in json.loads(output)['warnings']:
```

Comment:
can you add a docstring to explain what you are doing.

Reply:
I don't think it's needed. Just reading the JSON stdout call from `dodgy` and following what https://api.coala.io/en/latest/Developers/Linter_Bears_Advanced.html#custom-processing-functions-with-process-output has. Do you still want me to add it?

Reply:
yup not everyones gonna read this.

Reply:
I've added the docstring.